fili-core/src/main/java/com/yahoo/bard/webservice/druid/model/response/DruidResultRow.java

```diff
@@ -0,0 +1,39 @@
+// Copyright 2017 Yahoo Inc.
+// Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms.
+package com.yahoo.bard.webservice.druid.model.response;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import org.joda.time.DateTime;
+import org.joda.time.DateTimeZone;
+
+/**
+ * A row of results in a Druid Response.
+ */
+public abstract class DruidResultRow {
+    @JsonIgnore
+    private final String timestamp;
+
+    /**
+     * Creates a DruidResultRow at the given timestamp.
+     *
+     * @param timestamp  The timestamp to be included in serialization of a response.
+     */
+    public DruidResultRow(DateTime timestamp) {
+        this.timestamp = timestamp.toDateTime(DateTimeZone.UTC).toString();
+    }
+
+    @JsonProperty
+    public String getTimestamp() {
+        return timestamp;
+    }
+
+    /**
+     * Adds a json key/value pair to the row.
+     *
+     * @param key  The key to be added.
+     * @param value  The value of the key.
+     */
+    public abstract void add(String key, Object value);
```

Comment:
This has to be able to store `String` and `Number` types. If you would like I could expand it to
```java
public abstract void add(String key, String value);
public abstract void add(String key, Number value);
```
but there is not a specific Type which can be associated with a row.